sessions/react-effects-and-localstorage/journal-app-localstorage/src/components/EntryForm/index.js

```diff
@@ -0,0 +1,33 @@
+import "./EntryForm.css";
+import Button from "../Button";
+
+function EntryForm({ onAddEntry }) {
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    onAddEntry(data);
+    event.target.reset();
+  }
+
+  return (
+    <form className="entry-form" onSubmit={handleSubmit}>
+      <div className="entry-form__title">New Entry</div>
+      <div className="entry-form__fields">
+        <div className="entry-form__field">
```

Comment:
I am wondering whether this element should rather be a `fieldset` instead of a `div`. The same is true for the second `<div className="entry-form__field">` below.

Reply:
I remember talking about this topic with @djfarly. You cannot apply flexbox styling to a `fieldset`

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset#styling_with_css

So, we need to use a `div` to apply the styles. We could have another `fieldset` wrapped around, the `div` but that would bloat the JSX

Reply:
yup - i tried that. Fieldsets are weird…